### DISCUSSION TITLE
Docs: Some improvements in the documentation

### DISCUSSION THREAD

**Discussion by jx-bamboo** [ddnexus/pagy#722] [Changes and Refactorings]
### 👀 Before submitting...

- [X] I searched through the [Documentation](https://ddnexus.github.io/pagy/)
- [X] I searched through the [Issues](https://github.com/ddnexus/pagy/issues)
- [X] I searched through the [Q&A](https://github.com/ddnexus/pagy/discussions/categories/q-a)

### Description

https://ddnexus.github.io/pagy/docs/api/stylesheets/
In this tutorial, I found some problems with css, such as rgb(229 231 235), maybe rgb(229, 231, 235) is more appropriate. Of course, there are multiple representations like this.

**Comment by benkoshy** [comment]
@jx-bamboo hi thank you for your msg. 

> In this tutorial, I found some problems with css, such as rgb(229 231 235), maybe rgb(229, 231, 235) is more appropriate. Of course, there are multiple representations like this.

- Please would you be able to elaborate on what the problem is?
- Background: from the sounds of it, it seems a matter of preference choice: if you prefer another color it can be replaced according to what you deem acceptable?

**Comment by jx-bamboo** [comment]
It's not about the color, don't you notice the missing comma (,)? Of course, if you think the absence of a comma is the general way to write it, that's up to you, I need to add a comma anyway.

**Reply by ddnexus** [reply]
That CSS is a "copy, paste and mess-it-up" from the conversion of the tailwind style coming from some tool that I don't remember. Is it illegal CSS?

It doesn't look so: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb , https://www.w3schools.com/cssref/func_rgb.php

**Comment by jx-bamboo** [comment]
![2024-07-17_15-43](https://github.com/user-attachments/assets/5feef809-07c0-410f-9f25-9cff22b99723)

Do you see what he wrote himself? I can't run in rails successfully, you can also watch the css tutorial.

**Reply by ddnexus** [reply]
That looks like the old syntax, according to this https://stackoverflow.com/a/66899037/10947970
